conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/NegationRuleSuite.scala

```diff
@@ -39,14 +39,19 @@ class NegationRuleSuite extends FunSuite with SparkTestBase with LoggerTestBase{
 
     assert(intercept[ValidationException] {
       NegationRuleInterpreter.validateInputField("dataset", schema, "date")
-    }.getMessage contains "field is not a numeric type")
+    }.getMessage contains "field is not a algebraic type")
   }
 
   test("Negation conformance rule should negate positive numeric values") {
     val inputDataset = NegationRuleSamples.Positive.data.toDS
     val expectedDataset = NegationRuleSamples.dataset
     val expectedJSON = NegationRuleSamples.Positive.conformedJSON
 
+//    inputDataset.show(false)
```

Comment:
Redundant commnt

Reply:
This is in general just garbage left there by me.